README.rst

```diff
@@ -0,0 +1,203 @@
+Required packages
+-----------------
+
+To run the PyPI software, you need Python 2.7+ and PostgreSQL
+
+
+Quick development setup
+-----------------------
+
+Make sure you read http://wiki.python.org/moin/CheeseShopDev#DevelopmentEnvironmentHints
+and you have a working PostgreSQL DB.
```

Comment:
For example something like `psql -c 'create database test_db;' -U postgres`. I'm not sure if that example is universal or not though.